deblur/test/data/readme.txt

```diff
@@ -0,0 +1,7 @@
+70_otus.fasta - the 70% representative set from greengenes 13.5
```

Comment:
fixed to gg 13.8 (same data since it's just different taxonomy info)

On Thu, Mar 17, 2016 at 4:41 AM, Evguenia Kopylova <notifications@github.com
> wrote:

> In deblur/test/data/readme.txt
> <https://github.com/biocore/deblur/pull/32#discussion_r56490905>:
>
> > @@ -0,0 +1,7 @@
> > +70_otus.fasta - the 70% representative set from greengenes 13.5
>
> not 13.8? Can we put the latest?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/deblur/pull/32/files/069a0cd16c523f104f10937cd3dd5fd5715c57bb#r56490905>
>